pkg/cmd/openshift-apiserver/server.go

```diff
@@ -12,8 +12,6 @@ import (
 	openshiftcontrolplanev1 "github.com/openshift/api/openshiftcontrolplane/v1"
 	"github.com/openshift/openshift-apiserver/pkg/cmd/openshift-apiserver/openshiftapiserver"
 	_ "k8s.io/kubernetes/pkg/client/metrics/prometheus" // for client metric registration
-
-	// _ "k8s.io/kubernetes/pkg/util/reflector/prometheus" // for reflector metric registration
```

Comment:
reflector metrics were removed because of a memory leak, https://github.com/kubernetes/kubernetes/pull/74636 and https://github.com/kubernetes/kubernetes/pull/75700

Reply:
I think that those weren't monitored, for example:

https://github.com/search?q=org%3Aopenshift+reflector_watch_duration_seconds&unscoped_q=reflector_watch_duration_seconds

https://github.com/search?q=org%3Aopenshift+reflector_watches_total&unscoped_q=reflector_watches_total